UI/src/mixins/viewer.js

```diff
@@ -27,7 +27,7 @@ export const Viewer = {
     },
     openOhif(queryparams) {
       const queryparamsString = httpoperations.getQueriesParameters(queryparams);
-      return `${process.env.VUE_APP_URL_VIEWER}/viewer${queryparamsString}`;
+      return `${process.env.VUE_APP_URL_VIEWER}${queryparamsString}`;
```

Comment:
This allows specifying the full URL, which means things other than the base OHIF can be used, either a specific data source (which is the main item here), or a specific mode to be used.